src/sandbox-client.ts

```diff
@@ -317,6 +333,10 @@ export class SandboxClient {
     // Always add the "sdk" tag to the sandbox, this is used to identify sandboxes created by the SDK.
     const tagsWithSdk = tags.includes("sdk") ? tags : [...tags, "sdk"];
 
+    // We always want to start the VM in this context as our intention is to connect immediately
+    // or return the session data to manually connect, for example in browser
+    const startOptions = startOptionsFromOpts(opts) || {};
```

Comment:
Ah, I think we cannot pass `start_options` always because of a hack I currently do.

For the snapshot builder, you can specify a cluster to create the snapshot in. To enable this, we call `create` explicitly and then call `start` separately with a custom header in there to tell which cluster to force. This is super hacky, but I think that's why we don't always start.

I think we should keep the `start_options` as is, for now, and then open a separate PR to change this behaviour so we can test a bit.

Reply:
Ah, I see... okay, I added a comment on this now.